build/localnet/manifests/cli-client.yaml

```diff
@@ -17,13 +17,13 @@ spec:
     spec:
       serviceAccountName: debug-client-account
       initContainers:
-        - name: wait-for-validator-001-pocket-validator
+        - name: wait-for-validator-001-pocket
```

Comment:
I'm going to have `GetSession` implemented soon, which means that we're going to be able to have a "dispatch node".

1. Do you think this would enable us to have a dispatch node rather than using validator001 for everything?
2. If so, @okdas, can you create a ticket for it?